Dockerfile

```diff
@@ -2,7 +2,7 @@
 FROM node:14
 
 # set working directory & user
-USER node
+# USER node
```

Comment:
Is this committed on purpose?

Reply:
It definitely is not supposed to be there, it's just to get the thing to run locally in docker, but I must've been careless when adding the files.

Reply:
Both should be fixed now in the latest commit.